yeat/tests/test_cli.py

```diff
@@ -120,3 +121,56 @@ def test_custom_downsample_input(
     assert df.iloc[12]["sample_contigs"] == num_contigs
     assert df.iloc[13]["sample_contigs"] == largest_contig
     assert df.iloc[14]["sample_contigs"] == total_len
+
+
+@pytest.mark.parametrize("coverage", [("-1"), ("0")])
+def test_invalid_custom_coverage_1(coverage):
+    arglist = [
+        data_file("megahit.cfg"),
+        data_file("short_reads_1.fastq.gz"),
+        data_file("short_reads_2.fastq.gz"),
+        "--coverage",
+        coverage,
+    ]
+    with pytest.raises(SystemExit) as e:
+        args = cli.get_parser().parse_args(arglist)
+    assert isinstance(e.value.__context__, argparse.ArgumentError)
+    assert f"{coverage} is not a positive integer" in e.value.__context__.message
+
+
+@pytest.mark.parametrize("coverage", [("string"), ("3.14")])
+def test_invalid_custom_coverage_2(coverage):
```

Comment:
Less of a concern for test function names, but instead of `_1` and `_2` you might consider including something like `_negative` and `_noninteger` in the function name.

Reply:
Thanks for the suggestion!